webapp/views.py

```diff
@@ -811,6 +811,8 @@ def marketo_submit():
         value = ", ".join(values)
         if value:
             form_fields[key] = value
+            if 'utm_content' in form_fields:
+                form_fields['utmcontent'] = form_fields.pop('utm_content')
```

Comment:
Can you change it to double quotes (") for consistency? Thanks

Reply:
@carkod changed, thank you for catching!